winit/Cargo.toml

```diff
@@ -22,8 +22,9 @@ version = "0.1.0-alpha"
 path = "../native"
 
 [dependencies.window_clipboard]
-git = "https://github.com/hecrj/window_clipboard"
-rev = "22c6dd6c04cd05d528029b50a30c56417cd4bebf"
+git = "https://github.com/simlay/window_clipboard"
+branch = "ios-unimplemented-stub"
+#rev = "22c6dd6c04cd05d528029b50a30c56417cd4bebf"
```

Comment:
I just merged https://github.com/hecrj/window_clipboard/pull/1, so we should be able to depend on my fork again.